src/librustc_middle/mir/interpret/error.rs

```diff
@@ -327,6 +327,22 @@ impl fmt::Display for CheckInAllocMsg {
     }
 }
 
+/// Details of an access to uninitialized bytes where it is not allowed.
+#[derive(Debug)]
+pub struct UninitBytesAccess {
+    /// Location of the original memory access.
+    pub access_ptr: Pointer,
+    /// Size of the original memory access.
+    pub access_size: Size,
+    /// Location of the first uninitialized byte that was accessed.
+    pub uninit_ptr: Pointer,
+    /// Number of consecutive uninitialized bytes that were accessed.
+    pub uninit_size: Size,
+}
+
+#[cfg(target_arch = "x86_64")]
+static_assert_size!(UndefinedBehaviorInfo<'_>, 32);
```

Comment:
Actually what I meant is doing this for `InterpError`. We don't really care about the individual variants.

Reply:
Whoops, got it